### ISSUE TITLE
Recon dialog: suggested types should be clickable (missing <label> DOM element)

### ISSUE THREAD

**Issue by wetneb** [OpenRefine/OpenRefine#6328]
In the recon dialog, the suggested types are not clickable: one needs to click the radio button next to them.
This is bad accessibility.

### Proposed solution

Enclose the type name and id in a `<label>` element associated to the radio button, so that one can click on the text as well.

### Alternatives considered
none

### Additional context
![image](https://github.com/OpenRefine/OpenRefine/assets/309908/bd9c096f-793e-4bfb-870e-cadaf362afb5)

**Comment by chelseaqnguyen** [comment]
hi! can i work on this?

**Comment by wetneb** [comment]
Sure, go ahead!

**Comment by Redeem-Grimm-Satoshi** [comment]
@wetneb I can't seem to find the directory where the items in the column are rendered  on the [Standard Service Panel ](https://github.com/OpenRefine/OpenRefine/blob/master/main/webapp/modules/core/scripts/reconciliation/standard-service-panel.html )as show below;

![Reco](https://github.com/OpenRefine/OpenRefine/assets/45304978/ec265afb-c59f-4d1d-9311-ae224862b2d9)

Did I miss something?

**Comment by wetneb** [comment]
Have you tried this? https://openrefine.org/docs/technical-reference/code-contributions#locating-the-relevant-part-of-the-code-to-change

**Comment by Redeem-Grimm-Satoshi** [comment]
Yes,  

Searched `"Reconcile each cell to an entity of one of these types"` and  `"core-recon/cell-type"`, 
Which got me to `standard-service-panel.js` and `standard-service-panel.html`

**Comment by tfmorris** [comment]
The list of items comes from the backend reconciliation service based on the data sent to it from the column being reconciled. The example above seems to be from the Wikidata reconciliation service.

**Comment by wetneb** [comment]
Nicely done @Redeem-Grimm-Satoshi, you found the correct file: `standard-service-panel.js`.
This is indeed where the suggested types are rendered. Perhaps search for "type" in that file?

**Comment by Redeem-Grimm-Satoshi** [comment]
Yes @wetneb , located the 'type'

![type](https://github.com/OpenRefine/OpenRefine/assets/45304978/8268316e-88a5-4f29-b9da-7a29f713fe88)

**Comment by Redeem-Grimm-Satoshi** [comment]
How it works;

https://github.com/OpenRefine/OpenRefine/assets/45304978/1f992e89-6586-4f22-9191-08f2e0ccc5c5
